public/js/index.js

```diff
@@ -62,9 +63,6 @@ function submitForm(e) {
 }
 
 function ClipBoard(result) {
-	navigator.clipboard.writeText(result);
-	$('#copied').slideToggle(200);
-	setTimeout(function () {
-		$('#copied').slideToggle(200);
-	}, 1000);
+	 navigator.clipboard.writeText(result);
+	document.getElementById('copy').innerText='Copied'
```

Comment:
Remove this change and make a separate PR for it. You shall also change the CSS to a lil bit darker respective to the current color of the btn.